src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,141 @@
+define([
+    // Change this to filer vs. filer.min if you need to debug Filer
+    "thirdparty/filer/dist/filer.min",
+],function(Filer){
+    "use strict";
+
+    function ProjectStats() {};
```

Comment:
This doesn't need to be a function, it should be an object that you extend with methods below:

```js
var ProjectStats = {};
```